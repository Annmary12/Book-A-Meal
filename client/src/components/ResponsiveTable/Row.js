import React from 'react';
import classnames from 'classnames';
import PropTypes from 'prop-types';
import styles from './responsive-table.scss';

/**
 * @param {Object} props
 *
 * @returns {JSX} React JSX
 */
const Row = (props) => {
  const classNames = classnames({
    [styles.row]: true,
    [styles.header]: props.header
  });

  return (
    <div className={classNames}>
      {props.children}
    </div>
  );
};

Row.propTypes = {
  children: PropTypes.node.isRequired,
  header: PropTypes.bool
};

Row.defaultProps = {
  header: false
};

export default Row;
