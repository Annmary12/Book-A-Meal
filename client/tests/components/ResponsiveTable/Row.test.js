import React from 'react';
import Row from '../../../src/components/ResponsiveTable/Row';

describe('ResponsiveTable Row component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Row />);

    expect(wrapper).toMatchSnapshot();
  })
})