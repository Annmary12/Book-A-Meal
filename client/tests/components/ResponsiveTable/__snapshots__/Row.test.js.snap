// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveTable Row component should match snapshot 1`] = `
<div
  className="row"
/>
`;
